test/controllers/organizations/pets_controller_test.rb

```diff
@@ -57,4 +58,33 @@ class Organizations::PetsControllerTest < ActionDispatch::IntegrationTest
     assert_equal Mime[:html], response.media_type
     assert_response :redirect
   end
+
+  test "POST default pet tasks are created when pet is created" do
+    post pets_path, params: post_params
+
+    assert_equal Pet.last.tasks.count, 1
+  end
+
+  private
+
+  def post_params
```

Comment:
This is used in one spot so I think you can put in the `test` block above

Reply:
I thought it might be useful for future tests. But moving params to a private method can be done in future if necessary.